contracts/CTKNToken.sol

```diff
@@ -0,0 +1,357 @@
+pragma solidity ^0.4.19;
+
+import 'zeppelin-solidity/contracts/token/ERC20/MintableToken.sol';
+import './ERC884Draft.sol';
+
+
+/**
+ *  An `ERC20` compatible Token that conforms to Delaware State Senate,
+ *  149th General Assembly, Senate Bill No. 69: An act to Amend Title 8
+ *  of the Delaware Code Relating to the General Corporation Law.
+ *
+ *  Implementation Details.
+ *
+ *  An implementation of this Token standard SHOULD provide the following:
+ *
+ *  `name` - for use by wallets and exchanges.
+ *  `symbol` - for use by wallets and exchanges.
+ *
+ *  In addition to the above the following optional `ERC20` function MUST be defined.
+ *
+ *  `decimals` — MUST return `0` as each Token represents a single Share and Shares are non-divisible.
+ *
+ *  @dev Ref https://github.com/ethereum/EIPs/pull/884
+ */
+contract CTKNToken is ERC884Draft, MintableToken {
+
+    bytes32 constant private ZERO_BYTES = bytes32(0);
+    address constant private ZERO_ADDRESS = address(0);
+
+    string public name = 'CO2KN, INC Shares (indivisible)';
+    string public symbol = 'CTKN';
+    uint public decimals = 0;
+
+    mapping(address => bytes32) private verified;
+    mapping(address => address) private cancellations;
+    mapping(address => uint256) private holderIndices;
+
+    address[] private stockholders;
+
+    modifier isVerifiedAddress(address addr) {
+        require(verified[addr] != ZERO_BYTES);
+        _;
+    }
+
+    modifier isStockholder(address addr) {
+        require(holderIndices[addr] != 0);
+        _;
+    }
+
+    modifier isNotStockholder(address addr) {
+        require(holderIndices[addr] == 0);
+        _;
+    }
+
+    modifier isNotCancelled(address addr) {
+        require(cancellations[addr] == ZERO_ADDRESS);
+        _;
+    }
+
+    /**
+     * As each token is minted it is added to the stockholders array.
+     * @param _to The address that will receive the minted tokens.
+     * @param _amount The amount of tokens to mint.
+     * @return A boolean that indicates if the operation was successful.
+     */
+    function mint(address _to, uint256 _amount)
+        public
+        onlyOwner
```

Comment:
We can later change this to onlyWallet for multisig if we wished.  Also, was wondering should we use CappedToken? Its mintable: https://github.com/OpenZeppelin/zeppelin-solidity/blob/master/contracts/token/ERC20/CappedToken.sol

Reply:
We don't have the need for a token `cap`